### ISSUE TITLE
Request: Support for Rayman Jungle/Fiesta Run files?

### ISSUE THREAD

**Issue by LolHacksRule** [byvar/ubi-canvas#1]
I really like the potential of the editor but will the files from Rayman Jungle and Fiesta Run be supported soon? Here are Rayman Jungle Run's base game's assets. http://mob.ubi.com/rjr/rjr.zip.

**Comment by byvar** [comment]
Hi! Support for Rayman Jungle Run and Fiesta Run is planned - both were included when generating the serialization code, so any file format differences between Origins/Jungle Run/Fiesta Run should already be supported. I'll do it when I have some time. :)

Note to self:
- [ ] Add modes for Jungle Run & Fiesta Run
- [ ] Support for reading known texture formats without UbiArt header (just dds? png and pvr files are also included, but they seem to be conversions of the dds files)
- [ ] Reading sprites xml files. Check if necessary

**Comment by LolHacksRule** [comment]
Just Dance files use KTAPE (Karaoke TAPE) and DTAPE (Dance/Dancer TAPE) files, can you add support for them? Some platforms have these as plaintext JSONs, other platforms have UbiArt's CRC serialization used.

**Comment by yunylas** [comment]
Hey there byvar! I'm a Just Dance modder for almost 2 years and I worked with a lot of different file styles from every Just Dance game. I even made my own scripts for encrypting/decrypting Just Dance files. If you need any kind of help, I'd love to help you improve this project.

My discord: yunyl#1917

**Comment by LolHacksRule** [comment]
Hate bumping but you thinking about this? Some JD modders would appreciate it.

**Comment by byvar** [comment]
I'm working on adding support for Just Dance, yes. I have a lot of work at the moment so it's going really slowly, but it is happening. We'll see if that includes KTAPE and DTAPE files, but probably. :)

**Comment by LolHacksRule** [comment]
Cool stuff.

**Comment by shudlife** [comment]
Hey, sorry to nag, but any updates on Just Dance support? Thanks!

**Comment by heyimyunyl** [comment]
> Hey, sorry to nag, but any updates on Just Dance support? Thanks!

He told me that he stopped working on the Just Dance support.

**Comment by shudlife** [comment]
> > Hey, sorry to nag, but any updates on Just Dance support? Thanks!
> 
> He told me that he stopped working on the Just Dance support.

Damn, that's a shame. Do you by any chance know any tools that could help with modding?

**Comment by heyimyunyl** [comment]
> > > Hey, sorry to nag, but any updates on Just Dance support? Thanks!
> > 
> > 
> > He told me that he stopped working on the Just Dance support.
> 
> Damn, that's a shame. Do you by any chance know any tools that could help with modding?

No, but we have a Discord server related to Just Dance modding.
https://discord.gg/uFrPBh7dKp
